stacks/syrah/syrah.yaml

```
@@ -707,6 +707,8 @@ external_packages:
             prefix: <external_prefix>/comsol/5.6/comsol56/multiphysics/
           - spec: comsol@6.0 %<core_compiler>
             prefix: <external_prefix>/comsol/6.0
+          - spec: comsol@6.2 %<core_compiler>
+            prefix: <external_prefix>/comsol/6.2
```

```suggestion
            prefix: <external_prefix>/comsol/6.2
     -  comsol@6.2
```
You might need to explicitly add the fact to install the new version